src/utils/index.js

```diff
@@ -8,6 +8,7 @@ module.exports.coordinates = require('./coordinates');
 module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
+module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
```

Comment:
OK. We can merge, and I can update it to have a mobile namespace. We have 5 now, and it'd be best to future proof where it lives since utils are public.